libs/core/src/lib/tabs/tab-utils/tab-directives.ts

```diff
@@ -81,23 +82,20 @@ export class TabIconDirective implements CssClassBuilder, OnInit {
      * The icon to include inside the element
      * See the icon page for the list of icons.
      */
-    private _icon: string;
-    @Input() set icon(icon: string) {
-        this._icon = icon;
-        this.buildComponentCssClass();
-    };
+    @Input()
+    icon: string
 
     /** @hidden */
     constructor(
         private _elementRef: ElementRef
-    ) {}
+    ) { }
 
     /** @hidden
      * Function runs when component is initialized
      * function should build component css class
      * function should build css style
      */
-    ngOnInit(): void {
+    public ngOnChanges(): void {
```

Comment:
Don't hate me :D You need to remove the `public` :)

Reply:
:) good point, I'll do that

Reply:
removed